server/helpers/youtube-dl/youtube-dl-cli.ts

```diff
@@ -139,7 +138,7 @@ export class YoutubeDLCLI {
     url: string
     latestVideosCount?: number
     processOptions: execa.NodeOptions
-  }): Promise<{ upload_date: string, webpage_url: string }[]> {
```

Comment:
Why did you remove the typing information?

Reply:
As it differs from what will be read, and it will be cast to a builder interface later in the pipe.

https://github.com/Chocobozzz/PeerTube/blob/437ac0c176525908114a9a98e82aeba0bb1bace5/server/helpers/youtube-dl/youtube-dl-list-builder.ts#L29

To be more similar to [`YoutubeDLCLI.getInfo()`](https://github.com/Chocobozzz/PeerTube/blob/437ac0c176525908114a9a98e82aeba0bb1bace5/server/helpers/youtube-dl/youtube-dl-cli.ts#L117) in behavior.

Should I modify both to reflect intermediate type info read by the builders?

Reply:
Added type decelerations for both.